chapter2/tests/t13/Dockerfile

```diff
@@ -1,17 +1,20 @@
+ARG NODE_VERSION=14-slim
+ARG NGINX_VERSION=1.22.0-alpine
+
 # syntax=docker/dockerfile:1.2 
-FROM --platform=${BUILDPLATFORM:-linux/amd64} node:14-slim AS builder
+FROM --platform=${BUILDPLATFORM:-linux/amd64} node:"${NODE_VERSION}" AS builder
```

Comment:
Let's remove the hardcode platforms from the dockerfile and pass them only as arguments

Reply:
> Let's remove the hardcode platforms from the dockerfile and pass them only as arguments

@udaltsovra 
Done